### ISSUE TITLE
ActiveSupport::Logger.broadcast method is dropped

### ISSUE THREAD

**Issue by kalashnikovisme** [anycable/anycable-rails#188]
**Ruby version:**
Ruby 3.2.2

**Rails version:**
7.1.0.rc1

**`anycable` gem version:**
1.4.2

**`anycable-rails` gem version:**
1.4.0

**`grpc` gem version:**
1.58.0-x86_64-linux

### What did you do?
Tried to run my application with new Rails 7.1.0.rc1

### What did you expect to happen?
Everything should work as before Rails upgrade.

### What actually happened?
Got this error

```
ecwidai-anycable-1  | undefined method `broadcast' for ActiveSupport::Logger:Class
ecwidai-anycable-1  | /usr/local/bundle/gems/anycable-rails-1.4.0/lib/anycable/rails/railtie.rb:31:in `block (2 levels) in <class:Railtie>'
ecwidai-anycable-1  | /usr/local/bundle/gems/anycable-core-1.4.1/lib/anycable/cli.rb:175:in `each'
ecwidai-anycable-1  | /usr/local/bundle/gems/anycable-core-1.4.1/lib/anycable/cli.rb:175:in `configure_server!'
ecwidai-anycable-1  | /usr/local/bundle/gems/anycable-core-1.4.1/lib/anycable/cli.rb:52:in `run'
ecwidai-anycable-1  | /usr/local/bundle/gems/anycable-core-1.4.1/bin/anycable:7:in `<top (required)>'
ecwidai-anycable-1  | /usr/local/bundle/bin/anycable:25:in `load'
ecwidai-anycable-1  | /usr/local/bundle/bin/anycable:25:in `<top (required)>'
```

According to [this commit](https://github.com/rails/rails/commit/1fbd812c47f7ba840390942e56d9b2ebbc260901) in Rails repo, `ActiveSupport::Logger.broadcast` method is dropped.

Sidekiq has the same issue. So, it was fixed [here](https://github.com/sidekiq/sidekiq/commit/a1a220f7939fc422c89706454b3bd6799dcf47d2).

I've implemented something like this in my fork. Look at [this](https://github.com/kalashnikovisme/anycable-rails/commit/47b26f53f8b7ba2b0e976c23fc82c4ce8ac75064). Looks like everything works. I'm testing it right now.

Didn't submit a PR cause I don't know how you manage Rails versions differences :slightly_smiling_face:

**Comment by palkan** [comment]
Thanks!

Fixed and released in 1.4.1

**Comment by MBM1607** [comment]
@palkan I am experiencing the same error even though I am using the latest version.

##### Ruby version: 3.2.2
##### Rails version: 7.1.2
#####  `anycable` Gem Version: 1.4.3
#####  `anycable-rails` Gem Version: 1.4.2

**Comment by palkan** [comment]
Hey @MBM1607 

Could you please share a stack trace?

**Comment by palkan** [comment]
@MBM1607 Thanks! How do you configure your logger? I guess, you're not using the default one. I pushed a fix to master, could you please check it?

**Comment by MBM1607** [comment]
@palkan The logger is the default one, but I am using [overmind](https://github.com/DarthSim/overmind) to run several processes at once, which is why the stack trace looks different.

Let me try with your latest fix, I will get back to you.

**Comment by MBM1607** [comment]
@palkan It is working flawlessly with the latest fix.

**Comment by palkan** [comment]
Thanks! Released in 1.4.3
